src/features/projects/hooks/useProjects.ts

```diff
@@ -21,8 +21,25 @@ export function useProjectsById(ids: string[]) {
 const seed = 0;
 export function useSearchProjects(filterOverride?: Partial<Filter>) {
   const { data: filter } = useFilter("projects");
+  const clientFilter = filter;
+  if (filter) {
+    Object.keys(filter).forEach((k) => {
+      if (k === "seed" && typeof filter[k] === "string")
+        clientFilter[k] = parseInt(filter[k]);
+      else if (k === "cursor" && typeof filter[k] === "string")
+        clientFilter[k] = parseInt(filter[k]);
+      else if (k === "limit" && typeof filter[k] === "string")
+        clientFilter[k] = parseInt(filter[k]);
+      else if (typeof filter[k] === "string" && filter[k] === "null")
+        clientFilter[k] = null;
+      else if (typeof filter[k] === "string" && filter[k] === "undefined")
+        clientFilter[k] = undefined;
```

Comment:
It's not possible to get these values because typeof string never be applied for null and undefined.

Reply:
add this because I got `"null"` as `search` param, and it will display nothing since there's no project named "null"